frontend/src/general/__tests__/DeleteDeviceButton.test.jsx

```diff
@@ -33,7 +35,7 @@ describe("DeleteButton", () => {
   beforeEach(() => {
     handleClick.mockImplementation((open) => [open, setOpen]);
     wrapper = Enzyme.shallow(
-      <DeleteDeviceButton deviceType="encoder" deleteId="1:10:111:999" />
+      <DeleteDeviceButton button deleteId="1:10:111:999" />
```

Comment:
I feel like there should also be a test where button is set as false and the test should test that it renders a MenuItem instead of a button.

Reply:
i thought that test would end up being more trouble than it's worth considering the code coverage is already at 100% but it only took like a minute LOL